src/components/menus/VMenu.js

```diff
@@ -135,7 +135,11 @@ export default {
       default: false
     },
     maxWidth: [Number, String],
-    minWidth: [Number, String]
+    minWidth: [Number, String],
+    zIndex: {
+      type: [Number, String],
+      default: null
```

Comment:
Default should be 6, see https://github.com/vuetifyjs/vuetify/blob/dev/src/stylus/components/_menus.styl#L25.

Also, update the stylus file and remove the defined z-index.